osu.Game/Utils/PowerStatus.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Utils
+{
+    /// <summary>
+    /// Provides access to the system's power status.
+    /// Currently implemented on iOS and Android only.
+    /// </summary>
+    public abstract class PowerStatus
+    {
+        /// <summary>
+        /// The maximum battery level considered as low, from 0 to 1.
+        /// </summary>
+        public abstract double BatteryCutoff { get; }
```

Comment:
Why is this a variable (and why is it different on different devices)? I'd much rather this be the same across the board.

Reply:
I made it a variable because the cutoff before the OS sends a notification depends on the device. Should the cutoff just be 25% across the board?

Reply:
Yeah, I'd say that's fine. And move it out of the provider class and into the place where the notification is created.